test/test_intensify/test_hyperband.py

```diff
@@ -0,0 +1,129 @@
+import unittest
+from nose.plugins.attrib import attr
+
+import logging
+import numpy as np
+import time
+
+from ConfigSpace import Configuration, ConfigurationSpace
+from ConfigSpace.hyperparameters import UniformIntegerHyperparameter
+
+from smac.scenario.scenario import Scenario
+from smac.tae.execute_func import ExecuteTAFuncDict
+from smac.intensification.hyperband import Hyperband
+from smac.runhistory.runhistory import RunHistory
+from smac.optimizer.objective import average_cost
+from smac.tae.execute_ta_run import StatusType
+from smac.stats.stats import Stats
+from smac.utils.io.traj_logging import TrajLogger
+
+
+def get_config_space():
+    cs = ConfigurationSpace()
+    cs.add_hyperparameter(UniformIntegerHyperparameter(name='a',
+                                                       lower=0,
+                                                       upper=100))
+    cs.add_hyperparameter(UniformIntegerHyperparameter(name='b',
+                                                       lower=0,
+                                                       upper=100))
+    return cs
+
+
+class TestHyperband(unittest.TestCase):
```

Comment:
Could you please add some tests for the init method?

Reply:
I added the tests as a part of the intensify test cases because testing the parameter updates would mean running intensify separately for that.